packages/suite/src/components/suite/CoinjoinStatusBar.tsx

```diff
@@ -0,0 +1,137 @@
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const ProgressPie = styled.div<{ progress: number }>`
+    width: 16px;
+    height: 16px;
+    margin-right: ${SPACING}px;
+    border-radius: 50%;
+    background: ${({ theme, progress }) =>
+        `conic-gradient(${theme.BG_GREEN} ${3.6 * progress}deg, ${theme.STROKE_GREY} 0)`};
+`;
+
+const StatusText = styled.span`
+    color: ${({ theme }) => theme.TYPE_GREEN};
+`;
+
+const NextRoundNote = styled.span`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+`;
+
+const Separator = styled.span`
+    margin: 0 ${SPACING / 2}px;
+`;
+
+const ViewButton = styled(Button)`
+    height: 20px;
+    margin-left: auto;
+`;
+
+enum MockCoinjoinPhase {
+    Starting,
+    Ongoing,
+    Error,
+}
+
+const STATUS_MESSAGES = {
+    [MockCoinjoinPhase.Starting]: 'Collecting inputs',
+    [MockCoinjoinPhase.Ongoing]: 'Joining coins',
+    [MockCoinjoinPhase.Error]: 'U poor now',
+};
+
+interface CoinjoinStatusBarProps {
+    accountKey: string;
+    session?: CoinjoinSession;
+}
+
+export const CoinjoinStatusBar = ({ accountKey, session }: CoinjoinStatusBarProps) => {
+    const accounts = useSelector(state => state.wallet.accounts);
+    const routerParams = useSelector(state => state.router.params) as WalletParams;
+
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+
+    const phase = MockCoinjoinPhase.Ongoing;
+    const progress = 36; //  current_phase / (total_phases / 100)
+
+    const { symbol, index, accountType } = accounts.find(
+        ({ key }) => key === accountKey,
+    ) as Account;
+
+    const {
+        symbol: symbolParam,
+        accountIndex: indexParam,
+        accountType: accountTypeParam,
+    } = routerParams || {};
+
+    const isOnAccounPage =
+        symbolParam === symbol && indexParam === index && accountTypeParam === accountType;
+
+    return (
+        <Container>
+            <ProgressPie progress={progress} />
+
+            <StatusText>
+                {STATUS_MESSAGES[(session?.phase as MockCoinjoinPhase) || phase]}
+
+                <Separator>•</Separator>
+
+                <Translation
+                    id="TR_COINJOIN_COUNTDOWN"
+                    values={{ rounds: '8' }} // TODO: How to get?
+                />
+            </StatusText>
+
+            {session?.deadline && (
+                <NextRoundNote>
+                    <Separator>•</Separator>
+
+                    <Translation
+                        id="TR_COINJOIN_ROUND_COUNTDOWN"
+                        values={{
+                            time: <CountdownTimer deadline={session.deadline} />,
+                        }}
+                    />
+                </NextRoundNote>
+            )}
+
+            {!isOnAccounPage && (
+                <ViewButton
+                    variant="tertiary"
+                    onClick={() =>
+                        goto('wallet-index', {
```

Comment:
We need to think about the state with multiple devices connected at the same time.

Reply:
553294c65ebc0f98cdd168c80c14542ad398e998